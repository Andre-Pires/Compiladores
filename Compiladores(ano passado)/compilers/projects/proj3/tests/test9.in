t1 = a
t2 = b
t3 = t1 + t2
a = t3
t1 = 0
t2 = f + 1
b = t1 + t2
t1 = a
t2 = b
t3 = t1 + t2
a = t3
t1 = 1 + 1
a = t1
b = 0
a = a
