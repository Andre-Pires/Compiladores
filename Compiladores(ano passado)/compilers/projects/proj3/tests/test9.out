 Number of Registers: 5
	r3 = ARP + 4
	r3 = *r3
	r3 = r3
	r2 = ARP + 8
	r2 = *r2
	r2 = r2
	r1 = r3 + r2
	r0 = r1
	r3 = 0
	r1 = ARP + 24
	r1 = *r1
	r2 = r1 + 1
	r1 = r3 + r2
	r3 = r0
	r2 = r1
	r4 = ARP + 4
	*r4 = r0
	r0 = r3 + r2
	r0 = r0
	r3 = 1 + 1
	r0 = r3
	r1 = 0
	r0 = r0
	r4 = ARP + 4
	*r4 = r0
	r4 = ARP + 8
	*r4 = r1
 Number Load Operations: 3
 Number Store Operations: 3
